documentation/versioning_policy.markdown

```diff
@@ -79,11 +79,11 @@ Commands can be versioned on an individual command basis, so we are generally fr
 
 Commands are primarily represented by a corresponding wire format. Wire formats are versioned along with the corresponding command.
 
-The reasons to trigger a new command version are more common, and in general if we aren’t sure it's easy enough to create another version anyway.
+Changes to an existing command version can be made if the change is largely deemed backward compatible (so long as the consumer is less strict about new unexpected parameters). For example, addition of optional parameters within a command or wire format will not require a new version. 
```

Comment:
I don't think that's true. What if we added another edge type [here](https://docs.puppet.com/puppetdb/4.1/api/wire_format/catalog_format_v8.html#data-type-relationship). I think the user would feel it's pretty reasonable that we start allowing that new edge time in v8, not requiring a version bump. I like keeping the wording as backward incompatible or something else similarly generic. I think just saying optional fields is too specific without adding much value.